### DISCUSSION TITLE
Join the WeChat group to help with translating materials to English

### DISCUSSION THREAD

**Discussion by LashaO** [PaddlePaddle/PaddleOCR#7367] [Q&A] [answered]
Hello! I want to contribute. How do I get started? I sent out a request for the Technical group but I am not getting accepted. Thanks!

**Comment by Evezerest** [comment]
Hi LashaO, thanks for your kindness!
I'm not sure whether we can contact via WeChat. If you are a slack user, you can join [PaddleOCR slack channel](https://join.slack.com/t/paddleocr/shared_invite/zt-1fk71cyw3-NXm_5jnix4dazo0k~CCQlw). The OCR i18n community is being built from scratch. so any advice about the ideal communication platform is also welcome.
Look forward to your participation! please let me know if you join the slack 😁

**Reply by yeu-github** [reply]
Can you share link please?

**Reply by Evezerest** [reply]
please refer the [new slack invitation](https://join.slack.com/t/paddleocr/shared_invite/zt-1hp6bcg4x-YDQZhRMhENMUT7iMS7~EIA)

**Reply by Evezerest** [reply]
if you want to join the WeChat group, please scan the QR code below
![image](https://user-images.githubusercontent.com/50011306/194802872-0bd7eaf4-dc2a-4804-a25b-949ca820b172.png)

**Reply by yeu-github** [reply]
> if you want to join the WeChat group, please scan the QR code below
> ![image](https://user-images.githubusercontent.com/50011306/194802872-0bd7eaf4-dc2a-4804-a25b-949ca820b172.png)
> 

I couldn't register wechat. I think I need reference for it, how can I solve it?

**Reply by Evezerest** [reply]
you need to download the WeChat app at https://www.wechat.com/en/, register it and use the 'Scan' function to scan the QR code above

**Reply by audo23** [reply]
Dear Evezerest： the QR code and the slack channel are both expired. Could you share new ones? Thank you!

**Reply by Evezerest** [reply]
https://join.slack.com/t/paddleocr/shared_invite/zt-1jjk36xdj-id4A7gMRMRjbeVlQKZI3_g the invitation will be expired on December 14th, please let me know if anyone else wants to join the channel

**Comment by Evezerest** [comment, marked as answer]
WeChat is a popular communication application in China but international developers may not be familiar with it, so we open a new slack channel. Two platforms are both available.

**Reply by Evezerest** [reply]
https://join.slack.com/t/paddleocr/shared_invite/zt-1jjk36xdj-id4A7gMRMRjbeVlQKZI3_g the slack invitation will be expired on December 14th, please let me know if anyone else wants to join the channel

**Reply by soham24** [reply]
Can I get a fresh invite? this slack link  seems expired

**Reply by enggabrielhabib** [reply]
Hello, could you post the wechat group again?

**Reply by soham24** [reply]
There is nothing in wechat group brother 
I mean no models

**Comment by LashaO** [comment]
Super. Thanks!

**Comment by enggabrielhabib** [comment]
Can anyone update the qr code of the wechat group?

**Reply by timothee-dunglas** [reply]
Same question for the slack group ! Thanks a lot

**Reply by soham24** [reply]
Slack group is also dead

**Reply by soham24** [reply]
@timothee-dunglas @enggabrielhabib  
slack group
https://join.slack.com/t/paddleocr/shared_invite/zt-1zv3m2vdx-JedWnrgAWMWCZo8bsWamzg

**Comment by enggabrielhabib** [comment]
I mean, it's okay if there is nothing, just ask some questions to the professional people there.
